includes/controllers/class-message.php

```diff
@@ -59,6 +59,13 @@ public function __construct( $args = [] ) {
 						'redirect' => [ $this, 'redirect_messages_view_page' ],
 						'action'   => [ $this, 'render_messages_view_page' ],
 					],
+
+                    'messages_block_user'  => [
+                        'method' => 'POST',
+                        'path'   => '/block-user',
```

Comment:
It's better to follow the REST naming based on resources:

/users/{user_id}/block